schema.graphql

```diff
@@ -95,63 +69,28 @@ type Market @entity {
   name: String!
   "The Protocol the Market is a part of"
   protocol: Protocol!
-  "All Accounts that have used this Market"
-  accounts: [AccountInMarket!]! @derivedFrom(field: "market")
-  "Asset deposited in the market as collateral"
-  depositAsset: Asset! # NOTE - this could be an array, but we should find a protocol that needs this first, before implementing like so
-  "Asset borrowed. CDP style is native asset - i.e. DAI. Aave style borrowed asset == deposit Asset"
-  borrowAsset: Asset! # NOTE - this could be an array, but we should find a protocol that needs this first, before implementing like so
+  "Underlying Asset"
+  asset: Asset!
```

Comment:
We were talking yesterday about how some protocols create a synthetic asset to be minted, such as in Maker. You supply ETH and you mint DAI to yourself. Previously i was saying call it `syntheticAsset`. but i think its a poor name now. Let's call it this for now:

```graphql
collateralBackedAsset: Asset
```
it will be null for compound